app/assets/main/react_components/helpers/UserActionsHelper.js

```diff
@@ -0,0 +1,31 @@
+export const formatedUserActions = (inVal) => {
+  return inVal.map((userActions) => ({
+    ...userActions,
+    id: userActions.id.toString(),
+  }));
+};
+
+export const acceptedUserActions = (inVal) => {
+  return formatedUserActions(inVal)
+    .filter((action) => action.status !== true)
+    .map((action) => ({ ...action }));
+};
+
+export const columnUserActions = (
+  acceptedList,
+  doneList,
```

Comment:
Is "done" a word we use on other places? Or is "performed" or "completed" better?